test/sol5/networkHelper.js

```diff
@@ -308,7 +309,7 @@ module.exports.removeReservesFromNetwork = async function (networkInstance, rese
 
 module.exports.genReserveID = genReserveID; 
 function genReserveID(reserveID, reserveAddress) {
-    return reserveID + reserveAddress.substring(2,20);
+    return reserveID + reserveAddress.substring(2,20) + "0".repeat(38);
```

Comment:
this fails proxy tests.
not sure why
reverted this change.

Reply:
After debugging, the problem here is from https://github.com/KyberNetwork/smart-contracts/blob/katalystRevamp/test/sol5/networkHelper.js#L333.